### PR TITLE
Issue #23

### PR DESCRIPTION
**Pull request by anuveyatsu**
* implemented vega-lite as a node module
* refactored DataPackageViewContainer to support vega-lite graph type
* refactored vegalite component to render single series chart
* refactored vegalite component to render multiple charts with single series

### CODE REVIEW

File: src/containers/DataPackageViewContainer.js [hunk 1]
```diff
@@ -12,11 +13,38 @@ export class DataPackageViewContainer extends React.Component {
...
+      }, {})
+    })
+    vlSpec.data.values = objects
+    vlSpec.encoding.x.field = view.state.group
+    vlSpec.encoding.y.field = view.state.series[0]
+    return vlSpec
+  }
+
   //Takes a view and generates Plotly layout.
   generateSpec(view) {
```
Comment (Fak3) [inline src/containers/DataPackageViewContainer.js]: Probably it is worth renaming to smth like generatePlotlySpec() now

File: src/containers/DataPackageViewContainer.js [hunk 2]
```diff
@@ -51,28 +79,47 @@ export class DataPackageViewContainer extends React.Component {
...
+      this.setState({
+        graphType: graphType,
+        data: data,
+        layout: layout
+      })
+    }
   }
 
   render() {
     //render PlotlyChart component and pass data, layout, and index
```
Comment (Fak3) [inline src/containers/DataPackageViewContainer.js]: This comment is incorrect now

**Review by Fak3 — CHANGES_REQUESTED** [review]

File: src/containers/DataPackageViewContainer.js (hunk 2 above)
Comment (Fak3) [reply src/containers/DataPackageViewContainer.js]: Can you please also fix the description of DataPackageViewContainer in the topmost comment?

**Review by Fak3 — CHANGES_REQUESTED** [review]
Can you please fix  test - "should generate spec for Plotly" it should use new method name.

And please launch tests to check that all tests pass

**Comment by anuveyatsu** [comment]
@Fak3 tests for containers are passimg. However, I'm working on fixing tests for chart and table rendering - we have been discussing that with @subhankarb. We will handle it as we replace Jest with Mocha.

**Review by Fak3 — APPROVED** [review]
LGTM

**Review by subhankarb — COMMENTED** [review]
LGTM

**Pull request merged**